kafka-aiven/vedtak/topic-vedtak-dev.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: kafka.nais.io/v1
+kind: Topic
+metadata:
+  name: aapen-ensligforsorger-fattet-vedtak
```

Comment:
Hva tenker dere om navnet? 
DVH-hendelse hadde allerede navnet `aapen-ensligforsorger-vedtak-*`.

Reply:
Enig, men tror det finnes en mulighet for att andre skal bruke denne topic - litt uklart hvilket behov de har akkurat nå, men burde være tilstrekkelig at de får en notis om att en person fått en endring i en stønad, sånn att de burde kunne bruke denne

Reply:
Kanskje vi kan kalle denne `aapen-ensligforsorger-iverksatt-vedtak` ? I stedet for`fattet`

Reply:
Enig, synes `aapen-ensligforsorger-iverksatt-vedtak` var et bedre navn 👍